kbase-extension/scss/partials/_appCell.scss

```diff
@@ -150,6 +150,11 @@
     display: none;
   }
 
+  [data-element=advanced-hidden-message] {
```

Comment:
:sneaky:

^^ imagine there was a great emoji there

Reply:
Yeah, probably not the best answer. I couldn't get a real path to that for some reason and forgot about doing a good BEM name. I'll fix that and be less sneaky.

Reply:
Ok, now less sneaky. I BEM'd that and added it to the bulk-import-cell partial.
Also, removed remaining inline styles, I think.